src/briefcase/cmdline.py

```diff
@@ -12,22 +24,52 @@
     UnsupportedCommandError,
 )
 
+COMMANDS = [
+    NewCommand,
+    DevCommand,
+    UpgradeCommand,
+    CreateCommand,
+    UpdateCommand,
+    OpenCommand,
+    BuildCommand,
+    RunCommand,
+    PackageCommand,
+    PublishCommand,
+]
```

Comment:
```suggestion
COMMANDS = [
    NewCommand,
    DevCommand,
    CreateCommand,
    OpenCommand,
    BuildCommand,
    UpdateCommand,
    RunCommand,
    PackageCommand,
    PublishCommand,
    UpgradeCommand,
]
```

This is a minor re-ordering to get commands into "normal use order" - i.e., the order in which you'll normally encounter them.

Reply:
Good call. Updated.